src/v0/sources/shopify/util.js

```diff
@@ -146,6 +147,41 @@ const getAnonymousIdFromDb = async (message, metricMetadata) => {
   return getAnonymousId(message);
 };
 
+/**
+ * This function sets the sessionId based on cart_token or id from the properties of message.
+ * @param {*} message
+ * @returns
+ */
+const getSessionIdFromDB = async (message, metricMetadata) => {
+  const cartToken = getCartToken(message);
+  const { event } = message;
+  if (!isDefinedAndNotNull(cartToken)) {
+    return null;
+  }
+  let sessionId;
+  try {
+    sessionId = await RedisDB.getVal(`${cartToken}`, 'sessionId');
+  } catch (e) {
+    logger.debug(`Time: ${new Date()} {{SHOPIFY::}} sessionId get call Failed due redis error ${e}`);
+
+    stats.increment('shopify_redis_failures', {
+      type: 'get',
+      ...metricMetadata,
+    });
+  }
+  stats.increment('shopify_redis_calls', {
+    type: 'get',
+    ...metricMetadata,
+  });
+  if (sessionId === null) {
```

Comment:
is you create a common isNull function or check if it's available then use it here?